constants.yml

```diff
@@ -18,3 +18,17 @@ encounterType:
 program:
   hearts:
     uuid: "6cceab45-756f-427b-b2da-0e469d4a87e0"
```

Comment:
As long as you don't need to change the uuids for any reason (I don't think you do?) you don't need these changes, the build will pick up the uuids from the config-pihemr.

Reply:
I think it is preferable to not have the layer of mysterious indirection. Nothing is gained by having config-ces rely on config-pihemr for these UUIDs.